core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -29,14 +29,14 @@ typedef float64 CellType_F64;
 #define BR_TABLE_TMP_BUF_LEN 32
 
 #if WASM_ENABLE_THREAD_MGR == 0
-#define get_linear_mem_size() linear_mem_size
+#define get_linear_mem_size() memory->memory_data_size
```

Comment:
Seems no need to modify this line, for single-threading, linear_mem_size is gotten at the beginning of function wasm_interp_call_func_bytecode, and is only updated after a function call and memory.grow opcode. Updating this line may lead to CI failure:
https://github.com/bytecodealliance/wasm-micro-runtime/actions/runs/7022252919/job/19106153525#step:10:2133

Reply:
I was getting a warning when doing that, that's why I replaced it:
```bash
wasm-micro-runtime/core/iwasm/interpreter/wasm_interp_fast.c:1183:39: warning: variable 'linear_mem_size' is uninitialized when used within its own initialization [-Wuninitialized]
    uint32 linear_mem_size = memory ? get_linear_mem_size() : 0;
           ~~~~~~~~~~~~~~~            ^~~~~~~~~~~~~~~~~~~~~
/Users/eloparco/dev/forks/wasm-micro-runtime/core/iwasm/interpreter/wasm_interp_fast.c:23:31: note: expanded from macro 'get_linear_mem_size'
#define get_linear_mem_size() linear_mem_size
                              ^~~~~~~~~~~~~~~
```

I can fix it by rewriting it as 
```c
uint32 linear_mem_size = 0;
if (memory)
    linear_mem_size = get_linear_mem_size();
```